db/version_builder.cc

```diff
@@ -1181,68 +1186,84 @@ class VersionBuilder::Rep {
       }
     }
 
-    // <file metadata, level>
-    std::vector<std::pair<FileMetaData*, int>> files_meta;
-    std::vector<Status> statuses;
-    for (int level = 0; level < num_levels_; level++) {
-      for (auto& file_meta_pair : levels_[level].added_files) {
-        auto* file_meta = file_meta_pair.second;
-        // If the file has been opened before, just skip it.
-        if (!file_meta->table_reader_handle) {
-          files_meta.emplace_back(file_meta, level);
-          statuses.emplace_back(Status::OK());
+    Status ret;
```

Comment:
I have shifted the code such that the lambda function is away from the threaded preload processing / loops.  Added a couple of minor comments.  My understanding of the unchecked status stuff is that the naked "Status ret;" will not be seen as unchecked since it is a return value.  The caller will need to perform the check.